services.yaml

```diff
@@ -174,12 +176,32 @@ paths:
           $ref: "#/responses/TBD"
         401:
           $ref: "#/responses/UnauthorizedError"
+  /services/{service_id}/rename:
```

Comment:
why do we need a POST for /rename? i think a POST on /{service_id} with name="new name" has a similar affect and is more general